Jenkinsfile

```diff
@@ -96,8 +96,7 @@ pipeline {
                         YETUS_ARGS+=("--basedir=${WORKSPACE}/${SOURCEDIR}")
 
                         # our project defaults come from a personality file
-                        # which will get loaded automatically by setting the project name
-                        YETUS_ARGS+=("--project=hadoop")
```

Comment:
Would you add `"--project=hadoop"`?

https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2226/2/console
The project name become 'unknown' and the name used in many places.

I'm +1 if that is addressed. Thank you @sunchao

Reply:
Sure. Will add back. Thanks